tools/configure.c

```diff
@@ -174,11 +174,13 @@ static const char *g_optfiles[] =
 
 static void show_usage(const char *progname, int exitcode)
 {
-  fprintf(stderr, "\nUSAGE: %s  [-d] [-b] [-f] [-l|m|c|u|g|n] [-a <app-dir>] <board-name>:<config-name>\n", progname);
+  fprintf(stderr, "\nUSAGE: %s  [-d] [-s] [-b] [-f] [-l|m|c|u|g|n] [-a <app-dir>] <board-name>:<config-name>\n", progname);
   fprintf(stderr, "\nUSAGE: %s  [-h]\n", progname);
```

Comment:
Removal of the -s option to change CONFIG_CXX_NEWLONG breaks the build test for me.  The type of newlong changes from compiler to compiler (since it does not depend on the compiler but only libs/libxx or on the standard C++ library).

Removing this option makes the NEWLONG setting hardcoded in the defconfig file and cannot be modified for different tool chains.

Can we please restore this option.  Currently the stm3240g-eval/knxwm configuration fails to build for me.  Others build OK.  The fix is NOT to change the defconfig files.  The fix is to restore the option to testbuild.sh.

See the other half of the fixe below.

Reply:
The logic need used to use the options was apparently removed elsewhere.  It just did the tweak to enable or disable CONFIG_CXX_NEWLONG depending upon the command line option.